From cc03419e709d53c48f8871d8bf81f9ccae62386f Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@foss.st.com>
Date: Mon, 27 May 2024 19:04:05 +0200
Subject: [PATCH 0868/1695] plat-stm32mp2: default enable CFG_DRIVERS_FIREWALL

Default enable the CFG_DRIVERS_FIREWALL switch that is used to enable
the support of the firewall framework.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 4f6e43456bb80b0fd7ba55589c2cd11d69b7e57c plat-stm32mp2: default enable CFG_DRIVERS_FIREWALL]
Change-Id: I8746a27081244d1cbc20bf67af8091950be8e5ef
Signed-off-by: Gatien Chevallier <gatien.chevallier@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/382948
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
Tested-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
---
 core/arch/arm/plat-stm32mp2/conf.mk | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/core/arch/arm/plat-stm32mp2/conf.mk b/core/arch/arm/plat-stm32mp2/conf.mk
index f4736a813..cda1e773d 100644
--- a/core/arch/arm/plat-stm32mp2/conf.mk
+++ b/core/arch/arm/plat-stm32mp2/conf.mk
@@ -280,3 +280,8 @@ endif
 ifeq ($(CFG_STM32_LPTIMER),y)
 $(call force,CFG_COUNTER_DRIVER,y)
 endif
+
+CFG_DRIVERS_FIREWALL ?= y
+ifeq ($(call cfg-one-enabled, CFG_STM32_RISAF CFG_STM32_RIFSC),y)
+$(call force,CFG_DRIVERS_FIREWALL,y)
+endif
-- 
2.47.2

